Flutter/cartesian-charts/axis-customization.md

```diff
@@ -1658,7 +1658,7 @@ The [`DateTimeCategoricalMultiLevelLabel`](https://pub.dev/documentation/syncfus
 
 ### Multi-level labels in logarithmic axis
 
-The [`LogarithmicMultiLevelLabel`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/LogarithmicMultiLevelLabel-class.html) is used to render multi-level labels in the logarithmic axis. Here the [`start`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/LogarithmicMultiLevelLabel/start.html) and [`end`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/LogarithmicMultiLevelLabel/end.html) properties need to be double values.
+The [`LogarithmicMultiLevelLabel`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/LogarithmicMultiLevelLabel-class.html) is used to render multi-level labels in the logarithmic axis. Here the [`start`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/NumericMultiLevelLabel/start.html) and [`end`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/NumericMultiLevelLabel/end.html) properties need to be double values.
```

Comment:
Use LogarthmicMultiLevelLabel

Reply:
@LavanyaGowtham2021,

I have ensured with above shared  LogarthmicMultiLevelLabel class name for start and end properties. It is not working. The previously added API link is correct.


https://github.com/user-attachments/assets/d8116dee-a212-4229-b774-42a4fdf0a119



Regards,
Lokesh P.